app/controllers/rewards_controller.rb

```diff
@@ -3,8 +3,13 @@
 class RewardsController < ApplicationController
   before_action :authenticate_employee!
 
+  REWARDS_PER_PAGE = 10
```

Comment:
I would consider putting this in the config file - example: https://bloggie.io/@kinopyo/rails-the-convenient-way-to-load-custom-configuration-yaml-file